src/unix/bsd/freebsdlike/freebsd/freebsd12/mod.rs

```diff
@@ -229,4 +235,4 @@ cfg_if! {
         mod b64;
         pub use self::b64::*;
     }
-}
+}
```

Comment:
nit: We usually put a line break on the last line, could you re-add a line break? Otherwise we will change it back-and-forth :)

Reply:
whoops. not sure how that happened.